src/data/logos.js

```diff
@@ -29,6 +29,11 @@ export const openSourceData = [
     logo: 'marianne',
     link: 'https://github.com/SocieteNumerique/pass-numerique',
   },
+  {
+    name: 'Bolt',
+    logo: 'bolt',
+    link: 'https://bolt.cm/',
```

Comment:
After 4.0 goes live, this will be the canonical domain, and currently has the v4-beta (which has API Platform built in)

```suggestion
    link: 'https://boltcms.io/',
```

Reply:
Thank you! I've changed it with the new logo.